ext_mongodb.php

```diff
@@ -21,6 +21,11 @@ public function getMessage() : string
 		return $this->message;
 	}
 
+	public function getInfo() : array
```

Comment:
@derickr: I noticed that `info` is set with the following code in `hippo_write_result_init()`:

```
if (first_item.exists(s_info)) {
    wce_obj->o_set(s_info, first_item[s_info], s_MongoDriverWriteConcernError_className);
} else {
    wce_obj->o_set(s_info, Variant(), s_MongoDriverWriteConcernError_className);
}
```

I assume `array` is not the proper annotation. Is an empty `Variant()` null? If so, should I use `mixed`?